source/blog/2017-04-27-rspec-3-6-has-been-released.md

```diff
@@ -0,0 +1,144 @@
+merged pull requests from over ZZZ different contributors!
+
+Thank you to everyone who helped make this release happen!
+
+## Notable Changes
+
+### Core: Errors outside examples now handled and formatted well
+
+In previous versions of RSpec, we allowed errors encountered while loading spec
+files or running `:suite` hooks to crash the ruby interpreter, giving you its
+default full-stacktrace output.  In RSpec 3.6, we now handle all errors that
+occur outside examples, and format them nicely including a filtered backtrace
+and a code snippet for the site of the error.  For example, an error in a
+`before(:suite)` hook is now formatted like this:
+
+<img width="479" alt="Errors outside example execution"
+src="/images/blog/errors_outside_example.png">
+
+Thanks to Jon Rowe for assisting with this feature.
+
+### Core: `config.fail_if_no_examples`
+
+As it currently stands RSpec will exit with code `0` indicating success if no
+examples are defined. This configuration option makes it possible to cause RSpec
+to exit with code `1` indicating failure. This is useful in CI environments, as
+it helps detect when you've misconfigured RSpec to look for specs in the wrong
+place or with the wrong pattern. When no specs are found and
+`config.fail_if_no_examples is set` we consider this to be an error as opposed
+to passing the suite run.
+
+~~~ ruby
+RSpec.configure do |config|
+  config.fail_if_no_examples = true
+end
+~~~
+
+A special thanks to Ewa Czechowska for getting this in to core.
+
+### Core: Output coloring is set automatically if RSpec is outputting to a tty.
+
+In past versions of RSpec, you were required to specify `--color` if you wanted
+output coloring, regardless of whether you were outputting to a terminal, a
+file, a CI system, or some other output location. Now, RSpec will automatically
+color output if it detects it is running in a TTY. You can still force coloring
+with `--color`, or if you are running in a TTY and explicitly do not want color,
+you can specify `--no-color` to disable this behavior.
+
+We thank Josh Justice for adding this behavior to RSpec.
+
+### Expectations: Scoped aliased and negative matchers
+
+In RSpec 3, we added `alias_matcher`, allowing users to 
+[alias matchers](http://rspec.info/blog/2014/01/new-in-rspec-3-composable-matchers/#matcher-aliases) 
+for better readability. In 3.1 we added the ability to define
+[http://rspec.info/blog/2014/09/rspec-3-1-has-been-released/#expectations-new-definenegatedmatcher-api](negated matchers)
+with the `define_negated_matcher` method.
+
+Before RSpec 3.6 when you called either of these methods the newly defined
+matchers were always defined at the global scope. With this feature you are able
+to invoke either `alias_matcher` or `define_negated_matcher` in the scope of an
+example group (`describe`, `context`, etc). When doing so the new matcher that
+you define will **only** be available for the duration of that example group:
```

Comment:
"Duration" is kinda a confusing way to define scope, because it makes it sound like the scope is clock-based.  Instead, how about:

> When doing so the newly defined matcher will **only** be available to examples in that example group and any nested groups.